### DISCUSSION TITLE
Suggestion: Account groups/Subaccounts

### DISCUSSION THREAD

**Discussion by SekoiaTree** [firefly-iii/firefly-iii#7601] [General]
So, here's my problem/usecase.

My asset accounts correspond exactly to my bank accounts. However, I would like to dedicate some money to piggy banks/savings, that *wouldn't* be counted in my actual balance, without actually transferring. This allows me to use recurring transactions to repeatedly add to a piggy bank, as well as having a clear "this is how much money you can actually spend" without going into the piggy banks page.

Therefore, I suggest adding groups to asset accounts, in the same way that there are groups for rules and for piggy banks. Groups could also be looked at like a full account, but each account in it is an account of its own as well.

Alternatively, subaccounts, which are pretty self-explanatory. That way there's also a "default" asset account in the "group".

**Comment by JC5** [comment]
Although there will not be [sub accounts](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/#sub-categories-sub-budgets-sub-accounts), grouping of accounts is on my list.

**Comment by d-tork** [comment]
For what it's worth, I upvote this suggestion. While doing my taxes I would have liked to see asset accounts grouped according to whether they are tax-advantaged/retirement or not, investment vs. checking/savings, crypto vs fiat, etc. Plenty of ways to slice them so I would be open to grouping _or_ classification tags (would prefer tags if it could be easily kept separate from transaction tags!)

**Reply by fett327** [reply]
I would also find a "tag" field for accounts useful.  It would help me group or categorize them for certain rules and reports.
